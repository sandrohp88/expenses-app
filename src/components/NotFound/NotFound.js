import React from 'react'

const NotFound = () => <div>NotFound component</div>
export { NotFound }
