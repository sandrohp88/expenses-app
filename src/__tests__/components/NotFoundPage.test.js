import React from 'react'
import { shallow } from 'enzyme'
import { NotFound } from '../../components/NotFound/NotFound'

test('render NotFound without crash', () => {
  const wrapper = shallow(<NotFound />)
  expect(wrapper).toMatchSnapshot()
})
