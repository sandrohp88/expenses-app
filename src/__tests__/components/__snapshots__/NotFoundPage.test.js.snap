// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render NotFound without crash 1`] = `
<div>
  NotFound component
</div>
`;
